bin/sdr

```diff
@@ -11,6 +11,21 @@ class CLI < Thor
     true
   end
 
+  no_commands do
+    def check_control_master(skip)
+      unless skip
```

Comment:
If the developer wants to skip this check, I wonder if the method should `return if skip`, just completely short-circuit the new method.

Reply:
I could be swayed either way, I was thinking this helps keep them from locking themselves out, but if they really want to... OK... LOL

Reply:
haha